### DISCUSSION TITLE
第5节课作业(5班)

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/Tutorial#327] [作业]
以下上传宝们的第五节课作业

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135583793?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135583793%22%2C%22source%22%3A%22m0_75085274%22%7D

**Comment by DreamBinary** [comment]
https://github.com/DreamBinary/InternLMCamp/tree/master/day05/work

**Comment by GitEasonXu** [comment]
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_5/coursework.md

**Comment by shengmouget** [comment]
[作业](https://github.com/shengmouget/LLM_Job?tab=readme-ov-file#%E7%AC%AC%E5%9B%9B%E6%AC%A1%E4%BD%9C%E4%B8%9A)

**Comment by JayceNing** [comment]
**基础作业：**
使用 LMDeploy 以本地对话、网页Gradio、API服务中的一种方式部署 InternLM-Chat-7B 模型，生成 300 字的小故事（需截图）
![image](https://github.com/InternLM/tutorial/assets/44694099/983813db-200b-4f9a-b2d0-eb1ae4df4296)

**Reply by JayceNing** [reply]
![image](https://github.com/InternLM/tutorial/assets/44694099/7922dd25-f12d-45f4-a0e1-9297756883b4)

**Comment by angellyao** [comment]
https://zhuanlan.zhihu.com/p/678433970

**Comment by Milkve** [comment]
[第五节作业](https://juejin.cn/spost/7325242594651848740)

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class5-HW/class5_HW.md

**Comment by sanbuphy** [comment]
https://zhuanlan.zhihu.com/p/680168238?
